### PR TITLE
Set tags of images back to latest

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
Fixed tags for installer for develop branch

### CODE REVIEW

File: installer/deploy/installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+kind: Job
+metadata:
+  name: installer
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - name: keptn-installer
+          image: keptn/installer:5228f98
```
Comment (agrimmer) [inline installer/deploy/installer.yaml]: Should the installer also point to the latest image?

File: installer/deploy/installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
```
Comment (agrimmer) [inline installer/deploy/installer.yaml]: I know each service has a deploy folder. But do we need this for the installer? The same files are located in installer/manifests/installer.

Comment (christian-kreuzberger-dtx) [reply installer/deploy/installer.yaml]: Actually those two files in installer/deploy/ should not have made it to this PR; that was a mistake. I'm removing them.

Comment (agrimmer) [reply installer/deploy/installer.yaml]: Ok, thanks!

**Pull request merged**